pipelines/S1/queries.log

```diff
@@ -92,26 +196,106 @@ SELECT T.jahr, T.energieprodukt, T.verbrauchskategorie, T.mittlerer_preis_rappen
+             name
+ Canton of Zurich
 
 Query Nr. 9
 -- Wie hoch waren die Preise für das günstigste verfügbare Stromprodukt für alle Unternehmenskategorien im Kanton Genf im Jahr 2012?
-SELECT T.verbrauchskategorie, MIN(T.mittlerer_preis_rappen_pro_kwh) FROM median_strompreis_per_kanton as T JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid WHERE S.name_de LIKE '%Genf%'     AND S.canton=TRUE     AND T.jahr=2012     AND T.verbrauchskategorie LIKE '%C_%' GROUP BY T.verbrauchskategorie ORDER BY T.mittlerer_preis_rappen_pro_kwh ASC;
- verbrauchskategorie MIN(T.mittlerer_preis_rappen_pro_kwh)
-                  C7                              14.10156
-                  C6                              15.00263
-                  C4                              17.42939
-                  C3                              18.31980
-                  C1                              19.95666
-                  C2                              20.06664
+SELECT T.verbrauchskategorie, MIN(T.mittlerer_preis_rappen_pro_kwh) FROM median_strompreis_per_kanton as T JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid WHERE S.name_de LIKE '%Genf%'     AND S.canton=TRUE     AND T.jahr=2012     AND T.verbrauchskategorie LIKE '%C_%' GROUP BY T.verbrauchskategorie, T.jahr, T.mittlerer_preis_rappen_pro_kwh ORDER BY T.mittlerer_preis_rappen_pro_kwh ASC;
+ verbrauchskategorie      min
+                  C7 14.10156
+                  C6 15.00263
+                  C4 17.42939
+                  C3 18.31980
+                  C1 19.95666
+                  C2 20.06664
 
 Query Nr. 10
 -- Wann haben die Strompreise für Haushalte mit einem Verbrauch von weniger als 7500 kWh pro Jahr im Kanton Zürich ihren Höchststand erreicht?
-SELECT T.jahr, T.verbrauchskategorie, MAX(T.mittlerer_preis_rappen_pro_kwh) FROM median_strompreis_per_kanton as T JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid WHERE S.name_de LIKE '%Zürich%'     AND S.canton=TRUE     AND T.verbrauchskategorie LIKE 'H_%'     AND T.verbrauchskategorie_grosse_kwh_pro_jahr < 7500 GROUP BY T.verbrauchskategorie ORDER BY T.mittlerer_preis_rappen_pro_kwh DESC;
- jahr verbrauchskategorie MAX(T.mittlerer_preis_rappen_pro_kwh)
- 2024                  H1                              40.04015
- 2024                  H2                              37.25029
- 2024                  H4                              35.15730
- 2024                  H3                              33.91238
+SELECT T.jahr, T.verbrauchskategorie, MAX(T.mittlerer_preis_rappen_pro_kwh) FROM median_strompreis_per_kanton as T JOIN spatial_unit as S on T.spatialunit_uid = S.spatialunit_uid WHERE S.name_de LIKE '%Zürich%'     AND S.canton=TRUE     AND T.verbrauchskategorie LIKE 'H_%'     AND T.verbrauchskategorie_grosse_kwh_pro_jahr < 7500 GROUP BY T.verbrauchskategorie, T.mittlerer_preis_rappen_pro_kwh, T.jahr ORDER BY T.mittlerer_preis_rappen_pro_kwh DESC;
```

Comment:
This query seems to have similar problems as the one above.

Reply:
I also had to make the query more complex for the `group by` to work